src/main/webapp/app/shared/modal/AddMutationModal.tsx

```diff
@@ -297,6 +297,13 @@ function AddMutationModal({
       warning: entityStatusAlteration.warning ? entityStatusAlteration.message : null,
       error: entityStatusAlteration.error ? entityStatusAlteration.message : null,
     };
+
+    // if the backend's response is different from the frontend response, set them equal to each other.
+    if (alteration?.alteration !== alterationName) {
```

Comment:
I believe alterationName is computed with other fields within the alteration model. For instance, if we have comment on the model, alterationName includes it. This change here will overwrite it.